src/sinks/aws_s_s/sink.rs

```diff
@@ -2,15 +2,6 @@ use super::{client::Client, request_builder::SSRequestBuilder, service::SSServic
 use crate::sinks::aws_s_s::retry::SSRetryLogic;
 use crate::sinks::prelude::*;
 
-#[derive(Clone, Copy, Debug, Default)]
-pub(crate) struct SqsSinkDefaultBatchSettings;
-
-impl SinkBatchSettings for SqsSinkDefaultBatchSettings {
```

Comment:
I'm wondering if we meant to use these 🤔

Reply:
Maybe we _meant_ to, but it is completely unused now and so causing a complaint.